specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkWatcher.json

```diff
@@ -2812,15 +2821,48 @@
       "properties": {
         "storageId": {
           "type": "string",
-          "description": "The ID of the storage account to save the packet capture session. Required if no local file path is provided."
+          "description": "The ID of the storage account to save the packet capture session. Required if no local file path or local path is provided."
```

Comment:
filePath

Reply:
updated the description, resolved the comment.